scripts/performance/main.py

```diff
@@ -39,6 +39,17 @@ class TimingHandler(logging.Handler):
     transformations.  It collects these records, extracts the timing
     information and adds it to the test data recorder.
 
+    Attributes:
+        _testRecord (OrderedDict): Current test's timing data collection
+
+    Example timing data structure:
```

Comment:
It would be better to make this a pair of `.. testcode::` / `.. testoutput::` blocks so that the doctests can ensure that the examples are correct.